### ISSUE TITLE
Add non-docker installation documentation

### ISSUE THREAD

**Issue by robled** [alexjustesen/speedtest-tracker#65]
For those of us not using Docker, being able to install this in a non-Docker environment would be nice (VM, systemd-nspawn, etc).

**Comment by alexjustesen** [comment]
Can you give me an idea of the environment you'd install on? This being a Laravel app we can leverage their docs + install the speedtest CLI

**Comment by alexdelprete** [comment]
There's [Homestead](https://github.com/laravel/homestead).

Technically, you can also install PHP+Nginx+MySQL etc. standalone, Docker is just a way to have a packaged environment in place very quickly, but you can always unpack a container and install all apps/services on the host machine. I would not recommend it though...

In case you can't use Docker, Homestead looks like the proper way to go.

**Comment by alexjustesen** [comment]
@alexdelprete homestead is for local development only so I wouldn't recommend that and Composer is just a dependency manager for PHP (like NPM for node or PIP for Python).

I'll keep this on the radar @robled but Docker is going to be the recommended approach as I'll never be able to account for every bare metal install scenario. Closing for now, but its been added to the features list.

**Comment by alexdelprete** [comment]
> Composer is just a dependency manager for PHP (like NPM for node or PIP for Python).

Yes, and if you want to configure a full stack on the host with some configuration-management for each app/service, you need something like that.

I was also thinking about ansible, maybe there's some "recipe"...

**Comment by robled** [comment]
> @alexdelprete homestead is for local development only so I wouldn't recommend that and Composer is just a dependency manager for PHP (like NPM for node or PIP for Python).
> 
> I'll keep this on the radar @robled but Docker is going to be the recommended approach as I'll never be able to account for every bare metal install scenario. Closing for now, but its been added to the features list.

Indeed, it would be difficult to account for every last Linux distro to install this on.  To clarify what I'm suggesting, have a look at this [README.md](https://github.com/klaussilveira/gitlist#requirements) for gitlist.  They provide a Docker installation method, but also list the generic dependencie requirements and steps needed to install the application (e.g. PHP 8.1, Apache, etc..).  

With this info, non-Docker users could write their own Ansible playbook to install the dependencies and configure the application, or even create distro packages themselves.

**Comment by siegero** [comment]
@alexjustesen Have you considered creating an installation based on lxc? I think that would make a nice addition to the docker container.

I can contribute in building such an installation based on Turnkey Linux (https://www.turnkeylinux.org). However I would need some input reg. which components/configurations/services etc. to pull from the docker container.

**Comment by alexjustesen** [comment]
@siegero I haven't and really because I have no way to test it or knowledge of how to build it.
